test/cpp/tensorexpr/test_loopnest.cpp

```diff
@@ -2904,6 +2904,252 @@ void testNormalizeAndSplitWithTail() {
   torch::jit::testing::FileCheck().run(expected_tail_ir, oss_tail.str());
 }
 
+void testFlattenSimpleLoopNest2D() {
+  KernelScope kernel_scope;
+
+  // Input IR:
+  //   for (int i = 0; i < 10; i++) {
+  //     for (int j = 0; j < 5; j++) {
+  //       A[i,j] = i * j;
+  //     }
+  //   }
+  BufHandle a_buf("A", {ExprHandle(10), ExprHandle(5)}, kInt);
```

Comment:
Nit: I think you could write simply `{10, 5}` instead of `{ExprHandle(10), ExprHandle(5)}`.

Reply:
Done.